OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -162,7 +162,8 @@ public interface INotifyChat { bool OnChat(string from, string message); }
 	public interface INotifyProduction { void UnitProduced(Actor self, Actor other, CPos exit); }
 	public interface INotifyOtherProduction { void UnitProducedByOther(Actor self, Actor producer, Actor produced, string productionType, TypeDictionary init); }
 	public interface INotifyDelivery { void IncomingDelivery(Actor self); void Delivered(Actor self); }
-	public interface INotifyDocking { void Docked(Actor self, Actor harvester); void Undocked(Actor self, Actor harvester); }
+	public interface INotifyDock { void Docked(Actor self, Actor dockable); void Undocked(Actor self, Actor dockable); }
+	public interface INotifyDockable { void Docked(Actor self, Actor dock); void Undocked(Actor self, Actor dock); }
```

Comment:
Please use this opportunity to add `RequireExplicitImplementationAttribute` to both new interfaces.

Reply:
That would make sense here